python_tests/python/build_and_run.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+cargo build
+cp ../../target/debug/librust_swig_test_python.so rust_swig_test_python.so
```

Comment:
hardcoded path to target is bad, and why we need this script,
there is ci_build_and_test.py ?